src/status_im/ui/components/colors.cljs

```diff
@@ -0,0 +1,5 @@
+(ns status-im.ui.components.colors)
+
+(def white "#4360df")
```

Comment:
if we started this, can we add descriptions to colours? where they used for example

Reply:
maybe we could get comments from design team? for example this colour for buttons, or for text, or for foreground etc

Reply:
:+1: WIP